### PR TITLE
closes #58: Added 'Send' button to UI, hooked up to send function

### LINKED ISSUE #58: Style iOS Chatroom View
Implementing React StyleSheets to make MVP look nicer. Specific goals I'd like to implement:

'Submit' button for chat
Text forms should move upwards when keyboard appears on display
Navigation bar

### PR DESCRIPTION
**Pull request by rkho**
Reducing scope of #58 by moving Navbar + Flex view into separate issues.

### CODE REVIEW

File: app/styles.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+var React = require('react-native');
+var { StyleSheet } = React;
+
+var styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    backgroundColor: '#ffffff',
```
Comment (arianf) [inline app/styles.js]: Leave out last comma

File: app/index.ios.js [hunk 2]
```diff
@@ -13,29 +15,56 @@ var {
   View,
   ListView,
   NavigatorIOS,
-  AlertIOS
+  AlertIOS,
+  TouchableHighlight,
```
Comment (pravipati) [inline app/index.ios.js:14]: leave out last comma

**Comment by arianf** [comment]
This looks good

**Comment by koziscool** [comment]
looks good in code, running on my machine, I get sequelize connect error, also red screen of death in emulator (not the same as the known bug).  Not sure if it's related to this pr.

**Comment by pravipati** [comment]
We just discussed this but in case it's helpful--I'm getting require issues when running the code locally because there are certain global elements that got commented out. 

screen shot 2015-04-10 at 3 11 06 pm

**Comment by pravipati** [comment]
:+1:

**Comment by koziscool** [comment]
pulled latest branch commit and red screen of death is gone for me.

**Pull request merged**